app/assets/v2/css/base.css

```diff
@@ -2,6 +2,16 @@
 
 /* ...because sometimes descending-specificity is a PIA to clean up */
 
+:root{
+  --brand-white: #ffffff;
+  --brand-pink: #f6146d;
+  --brand-green: #37ec9b;
+  --brand-light-violet: #bbafd0;
+  --brand-vivid-violet: #773ae0;
+  --brand-dark-violet: #130b88;
+  --brand-black: #0d023a;
+}
```

Comment:
probably wanna shift that over to https://github.com/gitcoinco/web/blob/master/app/assets/v2/css/colors.css
and if we are shifting to a hex codes to variables -> it would make sense to update references throughout the codebase